demisto_sdk/commands/test_content/tests/ServerContext_test.py

```diff
@@ -0,0 +1,76 @@
+import sys
```

Comment:
please keep filenames according to pep8 standard, for example `server_context_test.py`